licenses.txt

```diff
@@ -1,62 +1,419 @@
 
-Licenses
-========
+Managed manually
+----------------
 
-Backbone.js
-===========
-http://backbonejs.org/
+Library: backbone.js
+Site http://backbonejs.org/
 License: MIT
 
-
-Underscore
-==========
-http://underscorejs.org/
+Library: backbone-localstorage
+Site: https://github.com/jeromegn/Backbone.localStorage
 License: MIT
 
-
-Backbone.localStorage
-=====================
-https://github.com/jeromegn/Backbone.localStorage
+Library: backbone.mousetrap
+Site: https://github.com/elasticsales/backbone.mousetrap
 License: MIT
 
+Library: bootstrap3
+Site: http://getbootstrap.com/
+License: Apache Software License, Version 2
 
-Bootstrap
-=========
-http://getbootstrap.com/
-License: Apache
-
+Library: bootstrap-colorpicker
+Site: http://mjolnic.com/bootstrap-colorpicker/
+License: Apache Software License, Version 2
 
-json2js
-=======
-http://www.JSON.org/json2.js
+Library: json2js
+Site: http://www.JSON.org/json2.js
 License: Public Domain
 
+Library: jquery-1.7.2
+Site: https://jquery.com/
+License: MIT
 
-mousetrap.js
-============
-craig.is/killing/mice
-License: Apache (version 2.0)
+Library: jquery-ui
+Site: http://jqueryui.com/
+License: MIT and GNU GENERAL PUBLIC LICENSE Version 2
 
+Library: jsrender
```

Comment:
This can be removed. Haven't used it since almost "initial commit" in 2012 in my weblabs repo before this became "figure". 6284aa15b49db0c860d8829832774a5e9a212924